html/changelog.html

```diff
@@ -23,26 +23,27 @@
-			<font size='2'><b>Coders:</b> TLE, NEO, Errorage, muskets, veryinky, Skie, Noise, Numbers, Agouri, Noka, Urist McDorf, Uhangi, Darem, Mport, rastaf0, Doohl, Superxpdude, Rockdtben, ConstantA, Petethegoat, Kor, Polymorph, Carn, Nodrak, Donkie, Sieve, Giacom, Ikarrus, trubble_bass, Aranclanos, Cael_Aislinn, Cheridan, Intigracy, Malkevin, SuperSayu, DumpDavidson, Tastyfish, Yvar, Elo001, Fleure, ManeaterMildred, Miauw, MrPerson<br></font> 
-			<font size='2'><b>Spriters:</b> Agouri, Cheridan, Cruazy Guest, Deeaych, Deuryn, Matty406, Microwave, ShiftyEyesShady, Skie, Uhangi, Veyveyr, Petethegoat, Kor, Ricotez, Ausops, TankNut, Pewtershmitz, Firecage, Nienhaus2<br></font>
-			<font size='2'><b>Sounds:</b> Skie, Lasty/Vinyl<br></font>
-			<font size='2'><b>Main Testers:</b> Tenebrosity, Anyone who has submitted a bug to the issue tracker<br></font>
-			<font size='2'><b>Thanks to:</b> Baystation 12, /vg/station, NTstation, CDK Station devs, FacepunchStation, GoonStation devs, the original SpaceStation developers and Invisty for the title image.<br> Also a thanks to anybody who has contributed who is not listed here :( Ask to be added here on irc.</font>
-			<font size='2' color='red'><b><br>Have a bug to report?</b> Visit our <a href="https://github.com/tgstation/-tg-station/issues?labels=Bug&state=open">Issue Tracker</a>.<br></font>
-			<font size='2'>Please ensure that the bug has not already been reported and <u><a href="https://github.com/tgstation/-tg-station/issues/724"><b>use the template provided here</b></a></u>.</font>
+			<font size='2'><b>Current Project Maintainers:</b> deathride58, LetterJay, kevinz000<br></font>
+			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/Citadel-Station-13/Citadel-Station-13/graphs/contributors'>-Click Here-</a><br></font>
+			<font size='2'><b>Coders:</b> Poojawa, deathride58, kevinz000, Trilbyspaceclone, nik707, as well as the /tg/ crew. <br></font> 
+			<font size='2'><b>Spriters:</b> Michiyamenotehifunana, CameronWoof, and the folks who contributed to /tg/ station project. <br></font>
+			<font size='2'><b>Sounds:</b> /tg/ Sound crew.<br></font>
+			<font size='2'><b>Main Testers:</b> You poor saps, our player base. <br></font>
+			<font size='2'><b>Thanks to:</b> /tg/station, Baystation 12, /vg/station, NTstation, CDK Station devs, FacepunchStation, GoonStation devs, the original SpaceStation developers and Invisty for the title image.<br> Also a thanks to anybody who has contributed who is not listed here :( Ask to be added here on irc.</font>
+			<font size='2' color='red'><b><br>Have a bug to report?</b> Visit our <a href="https://github.com/Citadel-Station-13/Citadel-Station-13/issues">Issue Tracker</a>.<br></font>
+			<font size='2'>Please ensure that the bug has not already been reported upstream and <u><a href="https://github.com/tgstation/-tg-station/issues/724"><b>use the template provided here!</b></a></u>.</font>

```

Comment:
the `/-tg-station/` link is outdated, needs to be updated to `/tgstation/`

Reply:
these aren't outdated, actually. check 'em.